jmix-data/eclipselink/src/main/java/io/jmix/eclipselink/impl/mapping/UuidMappingProcessor.java

```diff
@@ -48,31 +46,28 @@ public class UuidMappingProcessor implements MappingProcessor {
 
     @Override
     public void process(MappingProcessorContext context) {
+        DatabasePlatform platform = context.getSession().getPlatform();
+        Preconditions.checkState(platform instanceof UuidMappingInfo,
```

Comment:
I am not sure that we should forbid using DatabasePlatform that does not implement our `UuidMappingInfo` interface. In theory users may want to use their own database that is not supported by Jmix, but is supported by EclipseLink. In this case, they will get an exception if they don't create a wrapper for the database platform. Maybe we just should not set the converter for UUID fields in case the database platform doesn't implement the `UuidMappingInfo` interface? Let's ask @knstvk , maybe he will remember what was an idea when he introduced the `UuidMappingInfo`.

Reply:
That was my first thought, but since we're relying on `UuidMappingInfo` it's probably better to throw an exception rather than converting to a String without any guarantee that it's a valid UUID.

Reply:
Rolled back `UuidConverter` as it is used as a fallback if the database platform doesn't implement `UuidMappingInfo`. Added more detailed JavaDoc.